src/components/PaymentTile.js

```diff
@@ -219,28 +265,27 @@ const PaymentTile = (props) => {
                                     color={`${!totalAllocated || totalAllocated > payment.amount ? 'red' : 'primary.main'}`}
                                 >
                                     <Typography variant='subtitle2'>
-                                        {`${totalAllocated} allocated`}
+                                        {`${project ? totalAllocatedContributors : totalAllocated} allocated`}
+                                        <br/>
+                                        {`to ${project ? numberOfFilteredContributorsAllocated : numberOfContributorsAllocated} ${numberOfContributorsAllocated == 1 ? 'contributor' : 'contributors'}`}
```

Comment:
This line is too long we should find a way to indent/refactor to use a variable above the return and render the variable